lib/assets/Assets.js

```diff
@@ -57,6 +59,7 @@ Assets.prototype.addSource = function(src, opts) {
   */
 Assets.prototype.export = function(src, opts) {
   var assets = new AssetsCollection();
+  this.moduleCollections.push(assets);
```

Comment:
Where is this used? If it is new public API, we should be sure to add appropriate tests so that it doesn't vanish.

Reply:
yes, this is new API to have access to the AssetsCollections in modules (for caching them)